digitalocean/database/resource_database_opensearch_config_test.go

```diff
@@ -20,17 +20,21 @@ func TestAccDigitalOceanDatabaseOpensearchConfig_Basic(t *testing.T) {
 			{
 				Config: fmt.Sprintf(testAccCheckDigitalOceanDatabaseOpensearchConfigConfigBasic, dbConfig, true, 10, "1"),
 				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("digitalocean_database_opensearch_config.foobar", "enable_security_audit", "true"),
 					resource.TestCheckResourceAttr("digitalocean_database_opensearch_config.foobar", "ism_enabled", "true"),
+					resource.TestCheckResourceAttr("digitalocean_database_opensearch_config.foobar", "ism_history_enabled", "true"),
```

Comment:
Apparently both `ism_enabled` and `ism_history_enabled` both must be true when setting the other `ism_history` settings or the API returns errors, e.g.

```
        Error: Error updating Opensearch configuration: PATCH https://api.digitalocean.com/v2/databases/d6b65b4f-954d-4ee0-a29b-29eb2b4d228f/config: 422 (request "7f2461f9-6c49-4291-a304-364f1d5eb740") invalid 'user_config': invalid input for opensearch: 'ism_history_enabled' is a required property
```

I've added `enable_security_audit` so that we have another bool value to test changes. This exposed the other bug.